tests/suites/test_suite_ssl.function

```diff
@@ -5198,5 +5229,263 @@ exit:
     mbedtls_debug_set_threshold(0);
     mbedtls_free(first_frag);
     PSA_DONE();
+/* END_CASE */
```

Comment:
The way this conflict got resolved is wrong I think - just needs a closing `}` by the looks of it.

I think this is because of the way git handles the conflict poorly.